terraform/implementation/main.tf

```diff
@@ -250,6 +259,54 @@ resource "helm_release" "agic" {
   ]
 }
 
+# Cert Manager
+
+resource "helm_release" "cert_manager" {
+  name             = "cert-manager"
+  repository       = "https://charts.jetstack.io"
+  chart            = "cert-manager"
+  namespace        = "cert-manager"
+  create_namespace = true
+  depends_on       = [azurerm_kubernetes_cluster.k8s]
+
+  set {
+    name  = "installCRDs"
+    value = true
+  }
+}
+
+resource "kubectl_manifest" "cert_manager_issuer" {
+  depends_on = [helm_release.cert_manager]
+
+  yaml_body = <<YAML
+apiVersion: cert-manager.io/v1
+kind: ClusterIssuer
+metadata:
+  name: letsencrypt-prod
+spec:
+  acme:
+    # You must replace this email address with your own.
```

Comment:
Should these comments still be kept?

Reply:
I don't guess we need them! I can remove